src/rules/no-large-snapshots.js

```diff
@@ -19,6 +48,31 @@ const reportOnViolation = (context, node) => {
     });
   }
 };
+const captureToMatchSnapshot = (context, node) => {
+  const snapshotName = `${titles.join(' ')} ${++snapshotCall}`; //getSnapshotName(context, node);
```

Comment:
I think you're counting up within a test file here, while Jest counts up within a test case?

Reply:
Nevermind just noticed the exit visitor. Should probably test both multiple snapshots in a test case and multiple snapshots in different test cases though

Reply:
@jeysal added a test w/ 2 large snapshots in 2 separate tests.  resolving this, but feel free to re-open it below if you see any issues.